### PR TITLE
DDF-2682 updated tables for neutral taxonomy in docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates tables for neutral taxonomy in DDF Docs.

#### Who is reviewing it?

@brendan-hofmann @emanns95 @AzGoalie 

#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)
[Data](https://github.com/orgs/codice/teams/data)

#### Choose 2 committers to review/merge the PR.

@jlcsmith
@kcwire
@pklinef
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

- check tables against wiki: https://codice.atlassian.net/wiki/display/DDF/Attribute+Taxonomy+and+Mappings
- builds successfully.

#### What are the relevant tickets?
[DDF-2682](https://codice.atlassian.net/browse/DDF-2682)
#### Screenshots (if appropriate)

[draft-documentation.pdf](https://github.com/codice/ddf/files/711864/draft-documentation.pdf)

#### Checklist:

- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by emanns95** [comment]
Should multi-valued attributes be specified as such? @brendan-hofmann

**Comment by brendan-hofmann** [comment]
That should be communicated in the datatype. Either `primitive_type` or `list of `primitive_type`. We could add an explanatory paragraph at the beginning regarding this convention, or we could rework the tables to make it more explicit as its own column. That could be a future enhancement, I don't want to impede this initial PR too much.

File: distribution/docs/src/main/resources/_contents/_data/history-attributes-table-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+version
+|Date
+| 
+| 
+
+|version.action
+|Internal attribute for the action associated with a
+history metacard
+|String
+|One of `Created`, `Created-Content`, `Updated`,
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_data/history-attributes-table-contents.adoc]: I'm guessing `deleted-content` does not make any sense?

File: distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+|security.access-groups
+|Attribute name for storing groups to enforce access
+controls upon
+|String
+|
+|
+
+|security.access-individuals
+|Attribute name for storing the email addresses of users to
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc]: Should we include an example to show how to list the email addresses or multi-valued attributes in general? Maybe I'm missing a note that says these should be comma-delimited.

**Comment by ricklarsen** [comment]
I think instead of adding a column for multivalued, it could be a good use of the `constraints` column to add `multivalued`. It's often empty, and even where it isn't it's kind of a constraint on what value you can have.

File: distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc]: I like the idea of adding the note, as there currently isn't one.

File: distribution/docs/src/main/resources/_contents/_data/contact-attributes-table-contents.adoc [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+[cols="1,2,1,1,1" options="header"]
+|===
+|Term
+|Definition
+|Datatype
+|Constraints
+|Example Value
+
+|contact.creator-name
+|The name of this metacard’s creator.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_data/contact-attributes-table-contents.adoc]: @brendan-hofmann do you know the answer to this one?

**Comment by shaundmorris** [comment]
@ricklarsen good idea, that makes sense to use `constraints`.

File: distribution/docs/src/main/resources/_contents/_data/contact-attributes-table-contents.adoc (hunk 3 above)
Comment (brendan-hofmann) [inline distribution/docs/src/main/resources/_contents/_data/contact-attributes-table-contents.adoc]: It's just a string, so functionally it makes little difference. We should still establish a convention. Given that these names are displayed verbatim in the UI, I favor something human-readable like `titles prefixes first middle last suffixes` though obviously some of those are optional.

File: distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc (hunk 2 above)
Comment (brendan-hofmann) [inline distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc]: They are not comma delimited internally. It is actually a serialized list of values. I realize this is confusing and contradictory to how they are often treated in the Admin UI.

**Comment by brendan-hofmann** [comment]
I like the constraints idea, though it might make more sense treat being single-valued as a constraint, since internally we treat all attributes as potentially multivalued.

**Pull request merged**